testutil/keeper/tokenomics.go

```diff
@@ -71,9 +72,20 @@ type TokenomicsModuleKeepers struct {
 	Codec *codec.ProtoCodec
 }
 
-// TokenomicsKeepersOpt is a function which receives and potentially modifies the context
-// and tokenomics keepers during construction of the aggregation.
-type TokenomicsModuleKeepersOpt func(context.Context, *TokenomicsModuleKeepers) context.Context
+// tokenomicsModuleKeepersConfig is a configuration struct for a TokenomicsModuleKeepers
```

Comment:
How strongly do you feel about this? I would like to make a couple of observations in case that impacts your rationale:

1. This is rooted in the `testutil` package.
1. This type is not exported.